.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
Linters are failing! Kindly ensure that:
- [ ] Linters are set up correctly for this repository.
- [ ] Linters result should be green. AKA: Linter should report NO errors.
- [ ] There should be no custom changes to linter config (as an attempt to cheat linter error reports).